docs/api-reference/definitions.md

```diff
@@ -1,18 +0,0 @@
-<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
```

Comment:
Sorry this was a manual change. Fixed it now.
Most probably, I wont need to do this. @caesarxuchao has a PR that will most probably be cherrypicked before this.